tooling/api/tsup.config.ts

```diff
@@ -1,34 +0,0 @@
-// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
-// SPDX-License-Identifier: Apache-2.0
-// SPDX-License-Identifier: MIT
-
-import { defineConfig } from 'tsup'
-
-export default defineConfig(() => [
-  {
-    entry: ['src/*.ts'],
-    outDir: 'dist',
-    format: ['esm', 'cjs'],
-    clean: true,
-    minify: true,
```

Comment:
we used to minify everything, but now the terser plugin is only applied on the bundle script, is there a reason for that?

Reply:
The goal of this PR was to simplify the npm package files to enhance the DX of modifying and applying patches inside node_modules, and minifing the output makes that significantly harder. This is also a softer approach than going full JSDoc